src/spikeinterface/extractors/cbin_ibl.py

```diff
@@ -27,9 +27,11 @@ class CompressedBinaryIblExtractor(BaseRecording):
     load_sync_channel : bool, default: False
         Load or not the last channel (sync).
         If not then the probe is loaded.
-    stream_name : str, default: "ap".
+    stream_name : {"ap", "lp"}, default: "ap".
```

Comment:
This is cool I don't know this syntax, it means `"ap" | "lp"`?

Reply:
It signifies that your have a finite set of options, https://numpydoc.readthedocs.io/en/latest/format.html#parameters
Numpy just puts the default first, but I still like having the default explicitly stated